benches/bench.rs

```diff
@@ -68,6 +68,19 @@ fn bench_parsing_debug_info(b: &mut test::Bencher) {
     });
 }
 
+#[bench]
+fn bench_parsing_debug_aranges(b: &mut test::Bencher) {
+    let debug_aranges = read_section("debug_aranges");
+    let debug_aranges = DebugAranges::<LittleEndian>::new(&debug_aranges);
+
+    b.iter(|| {
+        let mut aranges = debug_aranges.aranges();
+        while let Some(arange) = aranges.next_arange().expect("Should parse arange OK") {
+            // Not really anything else we can check right now.
```

Comment:
The preferred style is to prefix unused variables with `_` or just name them `_`, which tells the compiler to shut up about not using them.

I'll fix this in a follow up after merging.